pkg/components/components.go

```diff
@@ -961,6 +950,27 @@ func multusClusterRoles() []rbacv1.PolicyRule {
 	}
 }
 
+func primaryUDNClusterRoles() []rbacv1.PolicyRule {
+	return []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{
+				"k8s.cni.cncf.io",
+			},
+			Resources: []string{
+				"network-attachment-definitions",
+			},
+			Verbs: []string{
+				"get",
+				"list",
+				"watch",
```

Comment:
those arent just for UDN
so either keep them at the original place please
or rename the function for something more generic
or just keep as you did before this change (where all the 6 are outside) please

Reply:
suggestions for a generic name?

Reply:
sorry please see the EDIT, the option for generic name is wrong
we must keep those 3 outside please

Reply:
OK